### DISCUSSION TITLE
Add `optimization` tag to proposals

### DISCUSSION THREAD

**Discussion by me2beats** [godotengine/godot-proposals#5356] [General Discussions]
For example this proposal can have this tag
https://github.com/godotengine/godot-proposals/issues/5351

As well as occlusion, lods etc.

Optimization is one of most important parts of game development

**Comment by Calinou** [comment]
There are `usability` and `performance` labels on the main Godot repository. I suppose these could also be added to the godot-proposals repository.

**Comment by Calinou** [comment]
[`performance`](https://github.com/godotengine/godot-proposals/labels/performance) and [`usability`](https://github.com/godotengine/godot-proposals/labels/usability) labels were added to godot-proposals issues where relevant.

**Reply by me2beats** [reply]
Yay cool!
